src/Microsoft.ComponentDetection.Detectors/go/GoComponentDetector.cs

```diff
@@ -223,5 +254,10 @@ private bool TryCreateGoComponentFromRelationshipPart(string relationship, out G
             goComponent = new GoComponent(componentParts[0], componentParts[1]);
             return true;
         }
+
+        private bool IsGoCliManuallyEnabled()
+        {
+            return EnvVarService.DoesEnvironmentVariableExist("EnableGoCliScan");
```

Comment:
Can you create a constants class for environment variables? If we're going to create a lot of these, it'll be easier to keep track of.